client/src/components/auth/LoginModal.js

```diff
@@ -41,8 +40,6 @@ class LoginModal extends Component {
         this.setState({ msg: null });
       }
     }
-
-    // If authenticated, close modal
     if (this.state.modal) {
```

Comment:
Change nested if to
 if(this.state.modal && isAuthenticated){
 this.toggle();
}